package.json

```diff
@@ -1,8 +1,13 @@
 {
   "name": "puppeteer",
-  "version": "13.2.0-post",
+  "version": "13.2.0",
```

Comment:
don't remote -post

Reply:
This seems to error with post in `linux-main`

Reply:
hm, it should not. That is the part of our release process, and usually it works.

Reply:
do you have a link to the error? the main workflows seem to be alright https://github.com/puppeteer/puppeteer/runs/5099856140?check_suite_focus=true